google-cloud-bigtable/lib/google/cloud/bigtable/convert.rb

```diff
@@ -77,6 +77,18 @@ def time_to_timestamp time
 
           Google::Protobuf::Timestamp.new seconds: time.to_i, nanos: time.nsec
         end
+
+        ##
+        # Converts an Integer to 64-bit signed big-endian integer data.
+        # Returns a string argument unchanged.
+        #
+        # @param value [String, Integer]
+        # @return [String]
+        #
+        def integer_to_signed_be_64 value
```

Comment:
Should have unit tests for this in test/google/cloud/bigtable/convert_test.rb.

Reply:
Added.